src/coreclr/src/jit/indirectcalltransformer.cpp

```diff
@@ -8,8 +8,8 @@
 #endif
 
 // The IndirectCallTransformer transforms indirect calls that involve fat function
-// pointers or guarded devirtualization candidates. These transformations introduce
-// control flow and so can't easily be done in the importer.
+// pointers, guarded devirtualization candidates, or runtime lookup with dynamic dictionary expansion feature.
```

Comment:
We should rename this base class and file, but maybe not just yet.

I have some work in progress that extends this too. Plenty of time to bikeshed on a new name.